core/frontend/src/internal/render/CesiumCamera.ts

```diff
@@ -0,0 +1,87 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+
+import { Cartographic, EcefLocation, ViewDefinition3dProps } from "@itwin/core-common";
+import { Point3d, Range3d, Vector3d, YawPitchRollAngles } from "@itwin/core-geometry";
+
+/** Properties that define a Cesium frustum.
+ * A [perspective frustum](https://cesium.com/learn/cesiumjs/ref-doc/PerspectiveFrustum.html) requires an fov and an [orthographic frustum](https://cesium.com/learn/cesiumjs/ref-doc/OrthographicFrustum.html) requires a width.
+ * @internal
+ */
+interface CesiumFrustum {
```

Comment:
These interfaces contain properties that get fed into CesiumJS to create the relevant CesiumJS objects.

It could be an option to put these interfaces, and future interfaces that serve the same "interop" goal into a separate file than the file that references them.

Then, they can be named simpler, like just Camera and Frustum.

When imported into a file where the system to which they refer makes sense, they can get imported as something different - like import Frustum as CesiumFrustum.

This is just a possibility I thought of while looking at the code --- I'm not necessarily saying this is the way it should be.

Reply:
@eringram That sounds great - feels clearer to me.